lib/core/form/components/widgets/form.theme.scss

```diff
@@ -27,10 +26,15 @@ ul > li > form-field > .adf-focus {
     }
 
     &-label {
-        color: rgb(186, 186, 186);;
+        color: rgb(186, 186, 186);
     }
 
     &-invalid {
+        .mat-form-field-label,
+        .mat-form-field-required-marker,
+        .mat-input-element ::placeholder {
+            color: #f44336 !important;
+        }
 
         .mat-form-field-underline {
             background-color: #f44336 !important;
```

Comment:
var(--theme-background-color);

Reply:
i removed css changes in favor of implement InvalidStateErrorMatcher